internal/generator/suite.go

```diff
@@ -191,3 +193,68 @@ func (s *Suite) String() string {
 
 	return spaceRegex.ReplaceAllString(strings.TrimSpace(result.String()), "\n")
 }
+
+const bashSuiteTemplate = `
+function setup() {
+	{{ .Setup }}
+}
```

Comment:
I think it would be move convenient to have separate setup for dependencies and the main setup.

In my previous testing I found that interdomain DNS setup can be painfully slow.
So currently in performance testing we use the following pattern:
```bash
setup_metallb
setup_dns
setup_spire

setup_nsm_and_run_test_and_cleanup_nsm
setup_nsm_and_run_test_and_cleanup_nsm
setup_nsm_and_run_test_and_cleanup_nsm

cleanup_spire
cleanup_dns
cleanup_metallb
```

It would be nice to have an ability to continue using something like this with the generated bash scripts.